include/multipass/constants.h

```diff
@@ -71,6 +71,11 @@ constexpr auto petenv_default = "primary";
 constexpr auto timeout_exit_code = 5;
 
 constexpr auto authenticated_certs_dir = "authenticated-certs";
+
+constexpr auto driver_deprecation_warning_template =
+    "*** Warning! The {0} driver is deprecated and will be removed in an future release. ***\n\n"
```

Comment:
```suggestion
    "**Warning! The {0} driver is deprecated and will be removed in an future release.**\n\n"
```
To follow Markdown style